include/Core/Users/Modules/DeleteBPPendingUsersModule.php

```diff
@@ -0,0 +1,126 @@
+<?php
+
+namespace BulkWP\BulkDelete\Core\Users\Modules;
+
+use BulkWP\BulkDelete\Core\Users\UsersModule;
+
+defined( 'ABSPATH' ) || exit; // Exit if accessed directly.
+
+/**
+ * Bulk Delete Buddy Press pending users.  This module is shown only when BuddyPress plugin is
+ * installed and active.
+ *
+ * @since 6.2.0.
+ */
+class DeleteBPPendingUsersModule extends UsersModule {
+	/**
+	 * Initialize and setup variables.
+	 */
+	protected function initialize() {
+		$this->item_type     = 'users';
+		$this->field_slug    = 'bp_pending_user';
+		$this->meta_box_slug = 'bd_bp_users_by_meta';
+		$this->action        = 'delete_bp_pending_users';
+		$this->messages      = array(
+			'box_label'        => __( 'Delete Pending Users', 'bulk-delete' ),
+			'confirm_deletion' => __( 'Are you sure you want to delete all the Buddy Press pending users?', 'bulk-delete' ),
+			/* translators: 1 Number of users deleted */
+			'deleted_one'      => __( 'Deleted %d Buddy Press pending user', 'bulk-delete' ),
+			/* translators: 1 Number of users deleted */
+			'deleted_multiple' => __( 'Deleted %d Buddy Press pending users', 'bulk-delete' ),
+		);
+	}
+
+	/**
+	 * Render delete users box.
+	 */
+	public function render() {
+		$count = \BP_Signup::count_signups();
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\BP_Signup' in method 'render'.](https://app.codacy.com/manual/BulkWP/bulk-delete/pullRequest?prid=4872271)